### PR TITLE
Add recipe for validictory

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

**Comment by souravsingh** [comment]
Pinging @jamesturk for review on the recipe.

**Comment by jamesturk** [comment]
thanks, I don't know much about Conda but this looks fine to me- thanks!

File: recipes/validictory/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - python
+
+test:
+  imports:
+    - validictory
+    - validictory.tests
+
+about:
+  home: http://github.com/jamesturk/validictory
+  license: BSD
```
Comment (ocefpaf) [inline recipes/validictory/meta.yaml]: The license seems to be MIT and not BSD. See https://github.com/jamesturk/validictory/blob/master/setup.py#L16

**Comment by souravsingh** [comment]
@jamesturk Would you like to be added as a maintainer for the recipe?

**Comment by jamesturk** [comment]
I'm afraid I wouldn't be of much help, so no thank you.

**Pull request merged**